src/libraries/System.Diagnostics.Process/tests/ProcessStartInfoTests.Windows.cs

```diff
@@ -68,5 +68,47 @@ void Cleanup(string username, string _)
                 }
             }
         }
+
+        [ConditionalTheory(typeof(RemoteExecutor), nameof(RemoteExecutor.IsSupported))]
```

Comment:
CI is failing with:
```
System.Diagnostics.Tests.ProcessStartInfoTests.TestWindowStyle(windowStyle: Normal, useShellExecute: True) [FAIL]
  System.Threading.ThreadStartException : Thread failed to start.
  ---- System.PlatformNotSupportedException : The system does not support the STA thread apartment.
  Stack Trace:
       at System.Threading.Thread.StartInternal(ThreadHandle t, Int32 stackSize, Int32 priority, Char* pThreadName)
    /_/src/coreclr/System.Private.CoreLib/src/System/Threading/Thread.CoreCLR.cs(90,0): at System.Threading.Thread.StartCore()
    /_/src/libraries/System.Private.CoreLib/src/System/Threading/Thread.cs(255,0): at System.Threading.Thread.Start(Boolean captureContext, Boolean internalThread)
    /_/src/libraries/System.Private.CoreLib/src/System/Threading/Thread.cs(226,0): at System.Threading.Thread.Start()
    /_/src/libraries/System.Diagnostics.Process/src/System/Diagnostics/Process.Win32.cs(178,0): at System.Diagnostics.Process.ShellExecuteHelper.ShellExecuteOnSTAThread()
    /_/src/libraries/System.Diagnostics.Process/src/System/Diagnostics/Process.Win32.cs(74,0): at System.Diagnostics.Process.StartWithShellExecuteEx(ProcessStartInfo startInfo)
    /_/src/libraries/System.Diagnostics.Process/src/System/Diagnostics/Process.Win32.cs(25,0): at System.Diagnostics.Process.StartCore(ProcessStartInfo startInfo)
    /_/src/libraries/System.Diagnostics.Process/src/System/Diagnostics/Process.cs(1282,0): at System.Diagnostics.Process.Start()
    /_/src/libraries/System.Diagnostics.Process/tests/ProcessStartInfoTests.Windows.cs(108,0): at System.Diagnostics.Tests.ProcessStartInfoTests.TestWindowStyle(ProcessWindowStyle windowStyle, Boolean useShellExecute)
       at System.RuntimeMethodHandle.InvokeMethod(Object target, Void** arguments, Signature sig, Boolean isConstructor)
    /_/src/libraries/System.Private.CoreLib/src/System/Reflection/MethodInvoker.cs(59,0): at System.Reflection.MethodInvoker.Invoke(Object obj, IntPtr* args, BindingFlags invokeAttr)
```

Windows nano does not support the STA thread spartment so you want to use this condition instead:
```suggestion
        [ConditionalTheory(nameof(IsAdmin_IsNotNano_RemoteExecutorIsSupported))]
```

Reply:
Thanks for the fix here.